web/js/containers/share.js

```diff
@@ -342,7 +349,7 @@ class ShareLinkContainer extends Component {
     } = this.state;
 
     return (
-      <div className="share-body">
+      <div className={`share-body${activeTab === 'embed' ? ' share-body-tall' : ''}`}>
```

Comment:
I think this can be simplified since we don't actually need both classnames (share-body and/or share-body-tall) in any circumstance.

Perhaps: `${activeTab === 'embed' ? ' share-body-tall' : 'share-body'}`

Or :  ```{`share-body${activeTab === 'embed' ? '-tall' : ''}`}```

Feel free to accept or ignore this nit.